### PR TITLE
Mat 7194/add action center and delete icon

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-7192](https://jira.cms.gov/browse/MAT-7192)
(Optional) Related Tickets:

### Summary

added action center and delete button to library details screen

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/PageHeader/CqlLibraryActionCenter/CqlLibraryActionCenter.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+import { SpeedDial, SpeedDialAction } from "@mui/material";
+import DeleteOutlinedIcon from "@mui/icons-material/DeleteOutlined";
+import { MadieDiscardDialog } from "@madie/madie-design-system/dist/react";
+import { CqlLibrary } from "@madie/madie-models";
+import { blue, red } from "@mui/material/colors";
+import { RouteHandlerState, routeHandlerStore } from "@madie/madie-util";
+
+interface PropTypes {
+  canEdit: boolean;
+  library: CqlLibrary;
```
Comment (sb-cecilialiu) [inline src/components/PageHeader/CqlLibraryActionCenter/CqlLibraryActionCenter.tsx:11]: I would think the action center is designed only for one library? Do you know if it will be used in library list?

Comment (sb-prateekkeerthi) [reply src/components/PageHeader/CqlLibraryActionCenter/CqlLibraryActionCenter.tsx:11]: This Action center is not present at the cql library list level. it is for only showing the action center inside the library so there wouldn't be multiple selection of libraries

**Review by sb-cecilialiu — APPROVED** [review]
